examples/local_orion/diffusion/config_diffusion.yaml

```diff
@@ -0,0 +1,45 @@
+# general
+exp_name: example_experiment
+run_name: run_debug_delete_me
```

Comment:
I suggest removing the `run_name` variable. When you set it, all the outputs on Comet look like this:

<img width="1418" alt="Screenshot 2024-04-16 at 8 32 12 AM" src="https://github.com/mila-iqia/diffusion_for_multi_scale_molecular_dynamics/assets/6283214/0ccffd94-d250-42fa-b552-48e77c0c6463">

If it is not defined, the code automatically assigns the orion `trial_id` as the run name:
<img width="1255" alt="Screenshot 2024-04-16 at 8 42 04 AM" src="https://github.com/mila-iqia/diffusion_for_multi_scale_molecular_dynamics/assets/6283214/322cd4ca-337c-463b-bff4-f404de3e51cf">